contracts/HeapOrdering.sol

```diff
@@ -141,18 +141,22 @@ library HeapOrdering {
         Account memory initialAccount = getAccount(_heap, _rank);
         uint256 initialValue = initialAccount.value;
         Account memory childAccount;
```

Comment:
childToSwap ?

Reply:
I'm ok with it. Not changing it in this PR though because we are not sure we want to merge it

Reply:
See #50